src/Auth/Handler.php

```diff
@@ -44,7 +44,7 @@ class Handler
      *                              authenticators are configured.
      * @throws ValidationException  If authentication is attempted and fails
      */
-    public function requireAuthentication(HTTPRequest $request)
+    public function requireAuthentication(HTTPRequest $request): Member|false
```

Comment:
Not supported in PHP 7.4
```suggestion
    public function requireAuthentication(HTTPRequest $request)
```